lua/github_access.lua

```diff
@@ -105,21 +105,27 @@ end
 
 function check_github(auth_md5, username)
     ngx.log(ngx.STDERR, "checking github for " .. username .. " (" .. auth_md5  .. ")")
-    local res =  ngx.location.capture("/force_github_request/orgs/" .. github_api_company .. "/members/" .. username)
+
+    -- authenticate
+    --
+    local res = ngx.location.capture("/force_github_request/user")
```

Comment:
I double checked if we ask the user to create a token with `read:user` we do. All good, no announcement needed :D